src/components/ServiceArea.jsx

```diff
@@ -2,14 +2,22 @@ import React from 'react'
 
 const ServiceArea = () => {
   return (
-    <div className='bg-gray-100'>
-      <div className='px-6 pt-10'>
-        <h2 className='text-center mb-3 text-5xl md:text-6xl font-header'>Areas We Serve</h2>
-        <hr className='w-[180px] md:w-[720px] h-px mx-auto mt-2 mb-10 md:mb-2 bg-gray-500 border-0 rounded dark:bg-gray-700' />
-        <p className='text-xl my-10 text-left'>We are proudly located in Detroit, Michigan. We service the <span className='font-bold'>Detroit, Highland Park, Hamtramck,</span> and <span className='font-bold'>metro Detroit </span>area. Unsure if we serve your location? Give us a call!
-        </p>
+    <div className='flex flex-col md:flex-row'>
+      <div className='pt-10 md:flex-2 md:flex-shrink-1 md:flex-grow-0 md:w-1/2 justify-center items-center md:px-24 md:pt-16 md:pb-32'>
+        <h2 className='text-center mb-3 text-5xl md:text-6xl font-header'>Where We Serve</h2>
+        <img src='../images/dog.png' className='w-[100%] h-[200px] object-contain pt-16 mb-5 md:pt-10' />
+        <div className='text-sm md:mt-10 flex flex-col md:justify-between'>
+          <p className='mb-10 md:my-10 px-6 md:px-0'>We are proudly located in Detroit, Michigan. We service the Detroit, Highland Park, Hamtramck, and metro Detroit area.
+          </p>
+          <iframe src="https://www.google.com/maps/d/embed?mid=1DG3IBrt1W1CSnau3KX33eHbSTV1uHtU&ehbc=2E312F" className='md:hidden px-0 h-[263px]' aria-label='Service Area Map'></iframe>
```

Comment:
Question: Should an `iframe` use `aria-label` or `title` (or something else)? Let's make sure we're using the right attributes for the element.